### PR TITLE
Flipper

### PR DESCRIPTION
**Pull request by DanMyers300**
Added Image flipper component

### CODE REVIEW

File: src/components/flipper.tsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import styles from '@/styles/Flipper.module.scss'
+import Image from 'next/image'
+import { useState } from 'react';
+import { FlipperImages } from '@/data/ice-age-texas/flipper/flipperData'
```
Comment (scottdickerson) [inline src/components/flipper.tsx:4]: @DanMyers300 we need the specific `ice-age-texas` data load to happen inside the `pages/stories/ice-age-texas` and then be passed to the `flipper` component please.  You can see if you

File: src/components/flipper.tsx [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  };
+  return (
+    <div className={styles.flipperWrapper}>
+      {FlipperImages.map((image, index) => (
+        <div key={index} className={styles.imageWrapper} onClick={() => toggleIsOpened(index)}>
+          <Image
+            src={isOpened[index] ? image.opened : image.closed}
+            className={styles.flipper}
+            alt={`Flipper ${index + 1}`}
+            layout="responsive"
```
Comment (scottdickerson) [inline src/components/flipper.tsx:22]: this has been deprecated and replaced with the `fill` property. (You should see an indicator in VSCode of this.)

File: src/components/flipper.tsx [hunk 3]
```diff
@@ -0,0 +1,31 @@
+import styles from '@/styles/Flipper.module.scss'
+import Image from 'next/image'
+import { useState } from 'react';
+import { FlipperImages } from '@/data/ice-age-texas/flipper/flipperData'
+
+export default function Flipper() {
+  const [isOpened, setIsOpened] = useState(new Array(FlipperImages.length).fill(false));
+
+  const toggleIsOpened = (index:any) => {
+    const newIsOpened = [...isOpened];
```
Comment (scottdickerson) [inline src/components/flipper.tsx:10]: very good job here to clone this array before changing it.

File: src/components/flipper.tsx [hunk 4]
```diff
@@ -0,0 +1,31 @@
+import styles from '@/styles/Flipper.module.scss'
+import Image from 'next/image'
+import { useState } from 'react';
+import { FlipperImages } from '@/data/ice-age-texas/flipper/flipperData'
+
+export default function Flipper() {
+  const [isOpened, setIsOpened] = useState(new Array(FlipperImages.length).fill(false));
```
Comment (scottdickerson) [inline src/components/flipper.tsx:7]: nice work

**Review by scottdickerson — CHANGES_REQUESTED** [review]
a few small changes here.  Thank you!

File: src/styles/Flipper.module.scss [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+  position: absolute;
+  top: 5%;
+  padding: 0 1vw;
+  min-width: 100%;
+  font-size: 1.2vw;
+  font-weight: 500;
+}
+
+.textIsClosed {
+    font-size: 1.8vw;
```
Comment (scottdickerson) [inline src/styles/Flipper.module.scss:24]: Can you increase the font-size to totally fill the available space on different smaller screen widths with a media query please?  Do your best to make it as visible as possible. 
image

File: src/styles/Flipper.module.scss [hunk 6]
```diff
@@ -0,0 +1,37 @@
+.flipperWrapper {
+    display: flex;
+    position: relative;
+    align-items: center;
+    justify-content: center;
```
Comment (scottdickerson) [inline src/styles/Flipper.module.scss:5]: I think the aspect-ratio css style might be needed here, when I tried `iPad Air` in the inspector for instance, it got weird:
image

File: src/components/flipper.tsx (hunk 4 above)
Comment (DanMyers300) [reply src/components/flipper.tsx:7]: Thank you

File: src/components/flipper.tsx (hunk 2 above)
Comment (DanMyers300) [reply src/components/flipper.tsx]: just removed it, doesn't seem necessary

File: src/components/flipper.tsx (hunk 1 above)
Comment (DanMyers300) [reply src/components/flipper.tsx:4]: Okay so I can kinda conceptualize how this is going to work but I don't have the technical skills to pull it off. Need to do some more research into React props and how all of that works

File: src/styles/Flipper.module.scss (hunk 6 above)
Comment (DanMyers300) [reply src/styles/Flipper.module.scss:5]: found the problem, it was a misplaced height setting for smaller screens.

File: src/components/flipper.tsx (hunk 1 above)
Comment (DanMyers300) [reply src/components/flipper.tsx]: Done

**Comment by DanMyers300** [comment]
retry deployment

**Pull request merged**